src/components/common/button/index.ts

```diff
@@ -1 +1,2 @@
+export type { ButtonProps } from './button';
```

Comment:
you should not export types here. import them as `import {ButtonProps} from '@/components/generic/button/button'`